qa/integration-tests/src/test/java/io/camunda/zeebe/it/management/JobStreamEndpointIT.java

```diff
@@ -0,0 +1,215 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.it.management;
+
+import static io.restassured.RestAssured.given;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+
+import io.camunda.zeebe.broker.StandaloneBroker;
+import io.camunda.zeebe.broker.system.configuration.BrokerCfg;
+import io.camunda.zeebe.client.ZeebeClient;
+import io.camunda.zeebe.it.smoke.CollectorRegistryInitializer;
+import io.camunda.zeebe.it.smoke.RandomPortInitializer;
+import io.camunda.zeebe.qa.util.actuator.JobStreamActuator;
+import io.camunda.zeebe.shared.management.JobStreamEndpoints.RemoteJobStream;
+import io.camunda.zeebe.shared.management.JobStreamEndpoints.RemoteStreamId;
+import java.time.Duration;
+import org.agrona.CloseHelper;
+import org.assertj.core.api.InstanceOfAssertFactories;
+import org.awaitility.Awaitility;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT, classes = StandaloneBroker.class)
+@ContextConfiguration(
+    initializers = {RandomPortInitializer.class, CollectorRegistryInitializer.class})
+@ActiveProfiles("test")
+public class JobStreamEndpointIT {
```

Comment:
Since for testing the endpoint we don't need to run more than one node - we're not testing the job stream functionality after all - I figured I can use a Spring Boot integration test instead of a container based test.

Reply:
:+1: Nice. It would be also faster than using containers. We should also check if we can use this in other actuator tests.